build.sbt

```diff
@@ -18,31 +20,30 @@ inThisBuild(
         "john@degoes.net",
         url("http://degoes.net")
       )
-    )
+    ),
+    supportedScalaVersions     := Map(
+      (zioConfigJS / thisProject).value.id                       -> (zioConfigJS / crossScalaVersions).value,
+      (zioConfigJVM / thisProject).value.id                      -> (zioConfigJVM / crossScalaVersions).value,
+      (zioConfigNative / thisProject).value.id                   -> (zioConfigNative / crossScalaVersions).value,
+      (zioConfigAwsJVM / thisProject).value.id                   -> (zioConfigAwsJVM / crossScalaVersions).value,
+      (zioConfigCatsJVM / thisProject).value.id                  -> (zioConfigCatsJVM / crossScalaVersions).value,
+      (zioConfigDerivationJVM / thisProject).value.id            -> (zioConfigDerivationJVM / crossScalaVersions).value,
+      (zioConfigEnumeratumJVM / thisProject).value.id            -> (zioConfigEnumeratumJVM / crossScalaVersions).value,
+      (zioConfigMagnoliaJVM / thisProject).value.id              -> (zioConfigMagnoliaJVM / crossScalaVersions).value,
+      (zioConfigPureconfigJVM / thisProject).value.id            -> (zioConfigPureconfigJVM / crossScalaVersions).value,
+      (zioConfigRefinedJVM / thisProject).value.id               -> (zioConfigRefinedJVM / crossScalaVersions).value,
+      (zioConfigScalazJVM / thisProject).value.id                -> (zioConfigScalazJVM / crossScalaVersions).value,
+      (zioConfigTypesafeJVM / thisProject).value.id              -> (zioConfigTypesafeJVM / crossScalaVersions).value,
+      (zioConfigTypesafeMagnoliaTestsJVM / thisProject).value.id -> (zioConfigTypesafeMagnoliaTestsJVM / crossScalaVersions).value,
+      (zioConfigYamlJVM / thisProject).value.id                  -> (zioConfigYamlJVM / crossScalaVersions).value,
+      (zioConfigXmlJVM / thisProject).value.id                   -> (zioConfigXmlJVM / crossScalaVersions).value
+    ),
+    scalafixScalaBinaryVersion := ""
   )
 )
 
-addCommandAlias("fmt", "; scalafmtSbt; scalafmt; test:scalafmt")
-addCommandAlias("fix", "; all compile:scalafix test:scalafix; all scalafmtSbt scalafmtAll")
 addCommandAlias("compileAll", "; ++2.12.16; root2-12/compile; ++2.13.8!; root2-13/compile")
```

Comment:
I would delete this as well. CommandAlias can be a pain. I would either choose it as sbtSettings  that are not based on String. Even better is within ecosystem plugin

Reply:
Done!